source/index.html.md

```diff
@@ -270,20 +287,23 @@ A successful request returns the HTTP `201 Created` status code with the followi
 | ---| ---| ---| --- |
 | user\_id | string | QEaakM0qumTJd5tj | The Betable user\_id as returned from the [Create User](#create-user) method call. |
 
+<aside class="success">
+The <code>user_id</code> returned should be used in subsequent API calls where a <code>user_id</code> is required as part of the URI.
+</aside>
```

Comment:
moved this up from Update User as Create User is actually where it gets returned